source-code/core/app/src/createInlang.ts

```diff
@@ -136,6 +137,23 @@ export const createInlang = async (args: {
 
 		const query = createReactiveQuery(() => messages()!)
 
+		let debouncedSave = (_newMessages) => {
+			// Skip initial call for setup
```

Comment:
I would move the comment near `atBegin`

Reply:
@ivanhofer   but it does not explain atBegin, but why i reassign to debouncedSave in the first call...

Reply:
Ah, now I see what you do here 😅
Nice trick, but it really is intransparent when not deeply thinking about it.. Can we find another solution?

Reply:
ok what about now? its also not pretty until js has pipe operator but as simple as it gets.

Reply:
way better!